### DISCUSSION TITLE
PDFTK-Java not installing (DEBIAN 10)

### DISCUSSION THREAD

**Discussion by the-hotmann** [Homebrew/discussions#402] [Linux]
About the Formula: https://github.com/Homebrew/homebrew-core/blob/master/Formula/pdftk-java.rb

When I run:
`brew install pdftk-java`
like recommended here:
https://formulae.brew.sh/formula/pdftk-java
then it starts 
brewing it. But fails with:
```
Error: An exception occurred within a child process:
  TypeError: no implicit conversion of nil into String
```

On the [PAGE](https://formulae.brew.sh/formula/pdftk-java) the statistics tell me there is not a single failed build Error so I'm curious about if I did something wrong?

My System:
Debian 10

```
user@main:~$brew -v
Homebrew 2.7.1
Homebrew/linuxbrew-core (git revision 263937; last commit 2020-12-30)
```

**Comment by gromgit** [comment]
Your Homebrew seems fairly up-to-date, but just to be sure there isn't anything odd about your installation, what's the output of `brew config` and `brew doctor`?
